src/main/kotlin/no/nav/familie/ef/sak/brev/VedtaksbrevService.kt

```diff
@@ -46,6 +80,23 @@ class VedtaksbrevService(private val brevClient: BrevClient,
         return beslutterPdf.bytes
     }
 
+    fun lagFritekstbrev(frittståendeBrevDto: VedtaksbrevFritekstDto): ByteArray {
+        val ident = behandlingService.hentAktivIdent(
+                UUID.fromString(frittståendeBrevDto.behandlingId))
+        val navn = personopplysningerService.hentGjeldeneNavn(listOf(ident))
+        val request = FrittståendeBrevRequestDto(overskrift = frittståendeBrevDto.overskrift,
+                                                 avsnitt = frittståendeBrevDto.avsnitt,
+                                                 ident = ident,
+                                                 navn = navn[ident]!!,
+                                                 brevdato = LocalDate.now())
+        val vedtaksbrev = lagreEllerOppdaterVedtaksbrev(behandlingId = UUID.fromString(frittståendeBrevDto.behandlingId),
+                                                        brevrequest = objectMapper.writeValueAsString(request),
+                                                        brevmal = "fritekst",
```

Comment:
Kan vi kanskje sette `fritekst` som en const noe sted? 
Kanskje på samme sted som `Vedtaksbrev.erFritekstType()` er, som også bruker den samme strengen
Då blir det litt enklere å finne hva den har for kobling til andre deler i koden og er ikke, kalles ofte for "magic string"

Reply:
Fikset i a422031